modules/hub/pages/hub-api-usage.adoc

```diff
@@ -0,0 +1,7 @@
+[[hub-api-namespaces]]
+= Hub XMRPC API Usage
+
+hub-xmlrpc-api is a daemon, currently to be launched from the command line.
```

Comment:
I think if we are implementing a systemd solution we should change this

Reply:
We will change this, once systemd solution is done.

Reply:
```suggestion
`hub-xmlrpc-api` is a daemon to be launched from the command line.
```

(it will change when it changes)